### PR TITLE
bazel - added rules player back for docs site

### PR DESCRIPTION
**Pull request by mercillo**
This adds back in the stamp rule and gh_deploy from rules_player. Allowing us to publish docs

### Change Type (required)
Indicate the type of change your pull request is:

- [ ] `patch`
- [x] `minor`
- [ ] `major`

### Does your PR have any documentation updates?
- [ ] Updated docs
- [ ] No Update needed
- [ ] Unable to update docs

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -7,9 +7,8 @@ bazel_dep(name = "rules_player")
 
 archive_override(
   module_name = "rules_player",
-  strip_prefix = "rules_player-1.0.1",
-  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.0.1.tar.gz"],
-  integrity = "sha256-zIbxx1Bh/X4E8OwbPJz1nY9mOd2uAgmTZyf1Hs2mq+U="
+  strip_prefix = "rules_player-1.1.0",
+  urls = ["https://github.com/player-ui/rules_player/archive/refs/tags/v1.1.0.tar.gz"],
```
Comment (KetanReddy) [inline MODULE.bazel:11]: Any reason we dropped the integrity check?

File: docs/BUILD [hunk 2]
```diff
@@ -1,10 +1,11 @@
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@rules_player//distribution/tar:defs.bzl", "stamp_tar_files")
+load("@rules_player//gh-pages:defs.bzl","gh_pages")
 
 genrule(
   name = "full_site",
   srcs = [
     "//docs/site",
-    "//docs/storybook"
```
Comment (KetanReddy) [inline docs/BUILD:7]: Are we not publishing storybook?

File: tsconfig.json [hunk 3]
```diff
@@ -22,7 +22,5 @@
     "resolveJsonModule": true,
     "composite": true,
     "lib": ["DOM", "ES2020"],
-    "isolatedDeclarations": true,
```
Comment (KetanReddy) [inline tsconfig.json:25]: Why did these change?

File: MODULE.bazel (hunk 1 above)
Comment (mercillo) [reply MODULE.bazel:11]: i can pull in the integrity for 1.1.0!

**Review by hborawski — APPROVED** [review]

**Review by KetanReddy — APPROVED** [review]

**Pull request merged**